wallet/internal/txsizes/size.go

```diff
@@ -106,6 +106,39 @@ func EstimateSerializeSize(scriptSizes []int, txOuts []*wire.TxOut, changeScript
 		changeSize
 }
 
+// EstimateSerializeSizeFromScriptSizes returns a worst case serialize size
+// estimate for a signed transaction that spends a number of outputs with scripts
+// identical in size to input script sizes provided and generates outputs with
+// scripts identical in size to the provided output script sizes. The estimated
```

Comment:
this is really wordy and I feel it will be incomprehensible.  Perhaps better: "...that spends len(inputSizes) previous outputs and pays to len(outputSizes) outputs with scripts of the provided worst-case sizes."